src/main/webapp/i18n/de/exam.json

```diff
@@ -606,6 +610,30 @@
                     "numberParticipants": "Teilnehmende",
                     "variants": "Varianten"
                 }
+            },
+            "suspiciousBehavior": {
+                "title": "Verdächtiges Verhalten",
+                "examSessionDescription": "Bei jedem Eintritt in eine Klausur wird für den Studierenden eine Sitzung erstellt. Diese Sitzung ist die Kombination aus IP-Adresse, Sitzungsschlüssel, User Agent, Browser-Fingerabdruck, Browserinstanz-ID und einem dazugehörigen Zeitstempel.",
+                "suspiciousSessionDescription": "Eine Sitzung wird als verdächtig eingestuft, wenn sie eine der folgenden Kriterien erfüllt:",
+                "suspiciousSessionCriterionIpAddress": "Es gibt mehrere Sitzungen mit derselben IP-Adresse, die nicht derselben Studierendenklausur zugeordnet sind",
+                "suspiciousSessionCriterionBrowserFingerprint": "Es gibt mehrere Sitzungen mit demselben Browser-Fingerabdruck, die nicht derselben Studierendenklausur zugeordnet sind",
+                "suspiciousSessionCriterionUserAgent": "Es gibt mehrere Sitzungen mit demselben User-Agent, die nicht derselben Studierendenklausur zugeordnet sind",
```

Comment:
`User-Agent` might not be clear to non-tech Users.

Reply:
I see your point but we use this term already like that in the user interface without an explanation, so I assumed we expect the Artemis instructor to know it.

Reply:
I agree that the wording for "User Agent" is probably ok, but it should be written consistently (currently, it's "User-Agent" here and "User Agent" for the `examSessionDescription`)